src/platforms/javascript/common/sourcemaps/troubleshooting.mdx

```diff
@@ -2,9 +2,9 @@
 title: Troubleshooting
 redirect_from:
   - /platforms/javascript/sourcemaps/troubleshooting/
----
-
-Source maps can sometimes be tricky to get going. If you’re having trouble:
+notSupported:
+  - javascript.nextjs
+---Source maps can sometimes be tricky to get going. If you’re having trouble:
```

Comment:
```suggestion
---
Source maps can sometimes be tricky to get going. If you’re having trouble:
```

Reply:
oops good catch.. auto formatter broke it